src/ARMInterpreter.cpp

```diff
@@ -266,6 +266,22 @@ void A_MRC(ARM* cpu)
 
 void A_SVC(ARM* cpu)
 {
+    // Print from game. Execute `svc 0xFC` with the null-terminated string address in `r0`.
+    if ((cpu->CurInstr & 0xFF) == 0xFC && cpu->NDS.GetDebugPrint())
```

Comment:
I tried `((cpu->CurInstr >> 16) & 0xFF)` as suggested, but this didn't work in practice. The interrupt instruction (`swi 0xFC`) was compiled to the following:

```
   8:   ef0000fc        svc     0x000000fc
```